### ISSUE TITLE
Day production always start with Minus Solar?

### ISSUE THREAD

**Issue by martin22cph** [egguy/grott-home-assistant-add-on#56]
Everyday, my PV or Solar always start with minus produktion?

See picture...

![image](https://github.com/egguy/grott-home-assistant-add-on/assets/33297250/e35abe08-efea-4a63-94bd-7d115f82876a)

**Comment by martin22cph** [comment]
Again today it did the same? 

![image](https://github.com/egguy/grott-home-assistant-add-on/assets/33297250/68d49cd9-9b60-47c4-9d7e-ba717afc0e4d)

**Comment by martin22cph** [comment]
This was a error 40, myswelf. The minus starting the day, was the prior days production, as i need to select the right sensor in Home Assistent. Selecting Lifetime solar energy, fixed the daily starting minus. :)

**Comment by egguy** [comment]
Yes, the energy calculation in HA works with a constantly incrementing value. I will try to update the documentation.

If you want to contribute to the documentation, I would be glad to accept your contribution.
